css/properties/font-family.json

```diff
@@ -46,6 +46,59 @@
             "deprecated": false
           }
         },
+        "math": {
+          "__compat": {
+            "description": "<code>math</code>",
+            "support": {
+              "chrome": {
+                "version_added": "103",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#enable-experimental-web-platform-features",
+                    "value_to_set": "Enabled"
+                  }
+                ]
+              },
+              "chrome_android": {
+                "version_added": false
+              },
```

Comment:
```suggestion
              "chrome_android": "mirror",
```

Reply:
As explained on https://docs.google.com/document/d/1bvY_Npe2zLW_705KXdmecsH6P9I9wBMpHRsZ9CxWNO on Blink generally does not use font-family keywords for font selection (it only has a few special handlings) and internal font settings are not used. However parsing, serialization, etc works as expected. Note that this limitation is not specific to `font-family: math` though, so I guess let's merge this...